src/dlgConnectionProfiles.cpp

```diff
@@ -170,11 +170,28 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
 
     character_password_entry->addAction(mpAction_revealPassword, QLineEdit::TrailingPosition);
-    if (mudlet::self()->storingPasswordsSecurely()) {
-        character_password_entry->setToolTip(tr("Characters password, stored securely in the computer's credential manager"));
-    } else {
-        character_password_entry->setToolTip(tr("Characters password. Note that the password isn't encrypted in storage"));
-    }
+    character_password_entry->setToolTip(tr("<p>The secret text needed to log into the game in order to play this profile's "
+                                            "player; if given, it will be supplied on connection to enable automatic log-in "
+                                            "to the game.</p>"
+                                            "<p>For more advanced situations it can be sent via a function "
+                                            "<tt>sendCharacterPasssword()</tt> in a script. For security it is not possible for "
+                                            "the script or any installed package or module to find out what the password is. "
+                                            "Further details are available in the Wiki.</p>"
+                                            "<p>%1</p>",
+                                            // Intentional comment to separate arguments
+                                            "%1 is an additional paragraph that indicates whether the password is current stored "
+                                            "EITHER: securely but not portably in the system's 'credential' store OR: insecurely "
+                                            "but portably {can be moved to another system or OS}. Note that the 'tt' tags and "
+                                            "the Lua function called 'sendCharacterPassword()' are to be left untranslated.")
+                                          .arg(mudlet::self()->storingPasswordsSecurely()
+                                               ? tr("Characters password, stored securely in the computer's credential manager.",
+                                                    // Intentional comment to separate arguments)
+                                                    "Additional paragraph text inserted at end of password entry tooltip when secure storage is used.")
+                                               : tr("Note that the password is <b>not</b> currently encrypted in storage.",
+                                                    // Intentional comment to separate arguments)
+                                                    "Additional paragraph text inserted at end of password entry tooltip when portable storage is "
+                                                    "used - bold emphasis has been applied to the Engineering English word that stresses that the "
+                                                    "storage method is insecure!")));
```

Comment:
It is difficult to pick out all the details in your analysis in order to respond concisely. I will review what has been said but particularly I would contend about the following:

* Tooltip addendum B:
  ...
  Its comment for translators [**suggestion for remove of all**]:
  * Info on where this sentence is used
  * Info on when this sentence is used
  * Info on why emphasis has been added

I haven't checked recently but surely each translation is listed separately in CrowdIn? That being the case where one thing is inserted into another it can be important to know the context in which the translation is used - which is why I bothered with the first two and I was aware that we have been trying to reduce the HTML tag loading on the translators - by keeping them out of the strings they have to work with but that particular pair is needed to make the formatting of the Lua function distinct. 

{Although, of course in relation to some HTML tags we should be keeping them in for bold and italics as those need to be *removed* as I understand it for Syllable-based logographic (I was going to say ideographic but apparently that isn't the correct word for either variant of Chinese) writing scripts}.